tests/h/util/group_test.py

```diff
@@ -10,10 +10,9 @@
 class TestSplitGroupID(object):
     @pytest.mark.parametrize('groupid,authority_provided_id,authority', [
         ('group:flashbang@dingdong.com', 'flashbang', 'dingdong.com'),
-        ('group::ffff@dingdong.com', ':ffff', 'dingdong.com'),
-        ('group:\\f!orklift@sprongle.co', '\\f!orklift', 'sprongle.co'),
+        ('group:ffff@dingdong.com', 'ffff', 'dingdong.com'),
```

Comment:
Most of these changes here are because I centralized the groupid pattern and things are more consistent now.